src/Test/Perf/util/runner_util.csx

```diff
@@ -38,3 +40,134 @@ async Task<ScriptState<object>> RunFile(string fileName)
     return await state.ContinueWithAsync<object>(text, scriptOptions);
 }
 
+/// Gets all csx file recursively in a given directory
+IEnumerable<string> GetAllCsxRecursive(string directoryName)
```

Comment:
Besides ignoring skip dirs, how is this different than ```AllCsiRecursive``` above?  Won't we want to skip certain files?